.github/ISSUE_TEMPLATE/blank.md

```diff
@@ -0,0 +1,11 @@
+---
+name: New Issue
```

Comment:
```suggestion
name: A New Issue
```
Looks like the issue templates are sorted by name? If so, how do you feel about a hack like this to keep this as the first option?

Reply:
![img](https://media.giphy.com/media/MGaacoiAlAti0/giphy.gif)